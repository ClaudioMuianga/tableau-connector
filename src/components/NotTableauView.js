import React, { Component } from 'react'
import {Col, Grid} from 'react-bootstrap'
import './NotTableauView.css'

import sparkle from '../static/img/new-sparkle-logo.png'

class NotTableauView extends Component {

  render () {
    return (
      <Grid className='notTableauView'>
        <Col md={6} mdOffset={3} xs={10} xsOffset={1}>
          <img src={sparkle} alt='data.world sparkle logo'/>
          <h2>
          Welcome to data.world's Tableau web data connector.
          </h2>
          <p>Open this page from Tableau Desktop to connect a dataset.</p>
          <a href='https://help.data.world/support/solutions/articles/14000062187-tableau-data-world-data-connector' target='_blank'>Learn more about the data.world connector</a>
        </Col>
      </Grid>
    )
  }
}

export default NotTableauView
