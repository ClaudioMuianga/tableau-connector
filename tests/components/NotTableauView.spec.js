import renderer from 'react-test-renderer'

import React from 'react'
import ReactDOM from 'react-dom'
import NotTableauView from '../../src/components/NotTableauView'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<NotTableauView />, div)
})

it('renders form', () => {
  expect(renderer.create(<NotTableauView />).toJSON()).toMatchSnapshot()
})