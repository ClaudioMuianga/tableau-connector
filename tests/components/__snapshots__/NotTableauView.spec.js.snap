exports[`test renders form 1`] = `
<div
  className="notTableauView container">
  <div
    className="col-md-6 col-md-offset-3 col-xs-10 col-xs-offset-1">
    <img
      alt="data.world sparkle logo"
      src="new-sparkle-logo.png" />
    <h2>
      Welcome to data.world\'s Tableau web data connector.
    </h2>
    <p>
      Open this page from Tableau Desktop to connect a dataset.
    </p>
    <a
      href=""
      target="_blank">
      Learn more about the data.world connector
    </a>
  </div>
</div>
`;
